core/src/main/java/hudson/util/FormFieldValidator.java

```diff
@@ -622,9 +622,9 @@ protected void check() throws IOException, ServletException {
                     return;
                 }
                 
-                com.trilead.ssh2.crypto.Base64.decode(v.toCharArray());
+                java.util.Base64.getDecoder().decode(v);
```

Comment:
Could you clarify which method you want to hide? Is it `Decoder#decode(...)`? I used that since it's in the Java 8 API, however Jenkins also has a Base64 decoder in the remoting classes which should be visible here. Should I just be updating that to delegate to the new Java decoder with any relevant exception catching and wrapping, and then calling that class from here?